recipe.yml

```
@@ -0,0 +1,24 @@
+name: Chronos Frontend Image
+id: chronos-fe
+stages:
+  - id: build
+    base: node:lts
+    singlelayer: false
+    labels:
+      maintainer: Vanilla OS Contributors
+    entrypoint: ["npm", "run", "publish"]
```

```suggestion
    entrypoint: ["cd /app && npm run publish"]
```

Tested this change on my fork https://github.com/kbdharun/chronos-frontend/actions and locally with VOS docs Portainer server.